src/Components/Generator/RequestTypeChooser.js

```diff
@@ -2,7 +2,7 @@ import preact from 'preact';
 import t from 'Utility/i18n';
 import Radio from '../Radio';
 
-const VALID_REQUEST_TYPES = ['access', 'erasure', 'rectification', 'custom'];
+const VALID_REQUEST_TYPES = ['access', 'erasure', 'rectification', 'objection', 'custom'];
```

Comment:
Let's abstract this somewhere and also use it in here https://github.com/datenanfragen/website/pull/169/files#diff-038cd559a6195a66a3d95ed073da4eadR54